checklists/alerts.md

```diff
@@ -0,0 +1,54 @@
+# Alerts
+
+## Sentry
+
+- [ ] Exceptions that the system may raise - [How to create this alert](https://docs.sentry.io/workflow/notifications/alerts/)
+- [ ] Highly repetitive exceptions within a short time window - [How to create this alert](https://sentry.io/_/resources/customer-success/alert-rules/)
+
+## Papertrail
+
+- [ ] Error keyword - [How to create this alert](https://help.papertrailapp.com/kb/how-it-works/alerts/#create-alert)
+- [ ] Exception keyword - [How to create this alert](https://help.papertrailapp.com/kb/how-it-works/alerts/#create-alert)
+- [ ] Scheduled tasks within a time window (success/failure) - [How to create this alert](https://help.papertrailapp.com/kb/how-it-works/alerts/#inactivity-alerts)
+
+## New Relic
+
+- [ ] Response time (web) - [How to create this alert](https://docs.newrelic.com/docs/alerts/new-relic-alerts/defining-conditions/create-alert-conditions)
+- [ ] Database transaction time - [How to create this alert](https://docs.newrelic.com/docs/alerts/new-relic-alerts/defining-conditions/create-alert-conditions)
+- [ ] Error percentage - [How to create this alert](https://docs.newrelic.com/docs/alerts/new-relic-alerts/defining-conditions/create-alert-conditions)
+- [ ] Key transactions response time - [How to create this alert](https://docs.newrelic.com/docs/apm/transactions/key-transactions/view-key-transactions-alert-information)
+- [ ] Key transactions error percentage - [How to create this alert](https://docs.newrelic.com/docs/apm/transactions/key-transactions/view-key-transactions-alert-information)
```

Comment:
Which alerts to you recommend for each one above? Higher than threshold? Deviation from baseline? Etc...
I think we shouldn't say only: `[ ] Response time (web)`, but something like `[ ] Response time (web) - higher than 300ms`

Reply:
Do you think that we should provide values? Asking this because this usually tends to vary from project to project, and by giving a brief search on this, people usually don't suggest these constraints.

Reply:
After discussion: don't provide values. But specify what are you tracking. Like `[ ] Response time (web) - above some threshold`

Reply:
After discussion: don't provide values. But specify what are you tracking. Like `[ ] Response time (web) - above some threshold`

Reply:
Change all other lines

Reply:
Just changed and applied to some other places that might seem a bit shadowy what's track objective.